src/js/app.js

```diff
@@ -0,0 +1,30 @@
+var newSearch;
+var searchResults;
+
+$(document).ready(function() {
+  // $('#testButton').on('click', function(e) {
+  //   e.preventDefault();
+  //   alert('Button was tested, gov\'nuh!');
+  // });
+  $('#searchSubmit').on('click', function(e) {
+    e.preventDefault();
+    newSearch = $('#searchInput').val();
+    $.get('https://www.reddit.com/search.json', {
+      q: newSearch
+    }).done(function(data) {
+      searchResults = data;
+      console.log(data);
```

Comment:
Nice, data retrieved! Here's a possible trajectory for your next steps forward:

* Create a function that grabs and filters the image links, and call it inside the `.done` promise
* Once the links have been filtered, pass those links into another function that displays the images to the screen
  * You can play with `setInterval` here, or get all images to display on the page, then go from there.